packages/web/components/complex/add-conc-liquidity.tsx

```diff
@@ -80,6 +80,14 @@ export const AddConcLiquidity: FunctionComponent<
       poolId,
     });
 
+    const getMagmaUrl = (pool?: Pool) => {
+      if (!pool) return "";
+      const assetDenoms = pool.reserveCoins
+        .map((coin) => coin.currency.coinMinimalDenom)
```

Comment:
Consider encoding denoms, esp those containing slashes, by wrapping with `encodeURIComponent` call

Reply:
using https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams might be also good